admin/controllers/payload.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
```

Comment:
i would have preferred a more descriptive name such as `pupil-payload-viewer` or similar.  Not hung up on it though, and appreciate it may be a pain to change at this stage, along with the service and tests etc..

Reply:
Renamed to be descriptive.  Good call 👍